### DISCUSSION TITLE
My Contributions shows 0 in my public profile

### DISCUSSION THREAD

**Discussion by devanshsachdeva** [community/community#23588] [New to GitHub]
<p>when i login to github, it says 23 contributions, while when i use my public profile with some interface in any app, it shows 0 contributions…</p>
<p>see the attached screenshots for reference<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/b/4b71c578f33d719f5b27562b02f80765760756a4.png" data-download-href="/uploads/short-url/aLpyZiFcghxx6zVHJPqlnT839nC.png?dl=1" title="git" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/25130502/181078079-59c3da77-7e35-45c4-b8dc-4f2397b85025.png" alt="git" data-base62-sha1="aLpyZiFcghxx6zVHJPqlnT839nC" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4b71c578f33d719f5b27562b02f80765760756a4_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4b71c578f33d719f5b27562b02f80765760756a4_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4b71c578f33d719f5b27562b02f80765760756a4_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4b71c578f33d719f5b27562b02f80765760756a4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">git</span><span class="informations">1920×1080 247 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<p>I think it is private repo contributions, enable it in <code>Contribution settings</code> so that it will be shown publicly ( just the count )</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/e/1e0d1e62bca0b17bf5046b6750df6765a9ff9520.png" data-download-href="/uploads/short-url/4hQqHWm8VesGHy81wQm2LeM901G.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181078080-e5103659-3dfc-4519-8807-e8f6db8dbdab.png" alt="image" data-base62-sha1="4hQqHWm8VesGHy81wQm2LeM901G" width="690" height="184" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/e/1e0d1e62bca0b17bf5046b6750df6765a9ff9520_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1058×283 14.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/0/10a140145e1586b3c23840dda65c8c97a876e959.png" data-download-href="/uploads/short-url/2n77eGLBKz7Q3uVGpS485OX3jPr.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181078081-0ebb71b6-c794-4410-8726-3e6516caa1e4.png" alt="image" data-base62-sha1="2n77eGLBKz7Q3uVGpS485OX3jPr" width="690" height="213" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/0/10a140145e1586b3c23840dda65c8c97a876e959_2_690x213.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/0/10a140145e1586b3c23840dda65c8c97a876e959_2_1035x319.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/0/10a140145e1586b3c23840dda65c8c97a876e959.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/0/10a140145e1586b3c23840dda65c8c97a876e959_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1121×347 27.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by devanshsachdeva** [comment]
<p>Tried that, didn’t work :confused:</p>

**Comment by devanshsachdeva** [comment]
<p>Update: working after an hour :slight_smile:</p>
<p>Thanks for the help Man !!</p>
